Tests/SwiftDependencyGraphsTests/Insert/InsertVertex.swift

```diff
@@ -0,0 +1,176 @@
+import Testing
+
+@testable import DependencyGraphs
+
+@Suite("Inserting a new vertex with a unique ID and") struct InsertVertexSuccessfullyTests {
+
+  @Suite("a unique label into the graph") struct UniqueLabelTests {
```

Comment:
These are just nits where I find the sentences weird to read.

-----


IMHO moving "and" into the sub-suite names is less confusing when looking at them further down.

```suggestion
@Suite("Inserting a new vertex with a unique ID") struct InsertVertexSuccessfullyTests {

  @Suite("and a unique label") struct UniqueLabelTests {
```

More pronounced in second child suite which just says

    "an existing label into the graph"

which isn't a sentence :)

------

`"into the graph"` can be dropped in the context of this library where all you do is manipulate graphs.

Maybe rephrase the suites, Then also move the suite below into this, so you get:

- "Inserting a vertex"
    - "with a unique ID and unique label"
    - "with a unique ID and existing label"
    - "twice" (formerly: "Inserting a new vertex twice into the graph", where "new" in "new vertex" is rather confusing)
    - "with an existing ID and unique label"
    - "with an existing ID and existing label" (tbh I'm not sure if this is different from the 'twice' case)